Packs/DigitalGuardian/Integrations/DigitalGuardianARCEventCollector/DigitalGuardianARCEventCollector.py

```diff
@@ -111,17 +112,21 @@ def set_export_bookmark(self, export_profile: str) -> str:
         )
 
 
-def test_module(client: Client, export_profiles: list[str]) -> str:
+def test_module(client: Client, export_profiles: list[str], export_calls_per_fetch: int) -> str:
     """
-    Tests API connectivity and authentication and validates export profiles.
+    Tests API connectivity and authentication and validates configuration parameters.
     Args:
         client (Client): Digital Guardian client to use.
         export_profiles (list): List of export profile names.
+        export_calls_per_fetch (int): Number of API calls to export events.
     Returns:
-        str: 'ok' if connection to the service is successful, an error message otherwise.
+        str: 'ok' if valid parameters and connection to the service is successful, an error message otherwise.
     Raises:
         DemistoException | Exception: If request failed.
     """
+    if export_calls_per_fetch < 0 or export_calls_per_fetch > 4:
```

Comment:
We should consider just verify this is a positive number, and let the clients be responsible but it can be the next phase.

Reply:
Updated validation to verify the value is ≥ 0